lib/ansible/modules/windows/win_msi.ps1

```diff
@@ -37,6 +37,11 @@ if (($creates -ne $null) -and ($state -ne "absent") -and (Test-Path $creates)) {
     Exit-Json $result
 }
 
+if (-not (Test-Path $path))
+{
```

Comment:
Sorry to nitpick but are you able to put the braces in the same line as the if statement to keep the same as the rest of the file?